src/validateSchema.js

```diff
@@ -8,20 +8,30 @@ const getValidationErrors = (schema, model) => Object.keys(schema).reduce((acc,
   const value = model[key]
   const rules = schema[key]
 
+  const renderError = (condition, fallback) => {
+    let err;
+    try {
+      err = rules.formatError({ key, value, condition, rules, schema, model });
+    } catch (e) {
+      err = rules.formatError || fallback;
```

Comment:
I like this implementation a lot, it's simple and straightforward. Thanks!

However, question: what is the use case for reassigning `formatError` to `err `(to be used as `renderError`) if its first invocation threw? Wouldn't that make the return value a function, and thereby push a function into the list of errors?

Overall, I like the idea, but think the `try/catch` is superfluous. Would something like this suffice?

```js
const renderError = (condition, fallback) => {
  return typeof rules.formatError === 'function'
    ? rules.formatError({ key, value, condition, rules, schema, model })
    : fallback
}
```